### DISCUSSION TITLE
Open-Shell 4.4.170

### DISCUSSION THREAD

**Discussion by tallman69** [Open-Shell/Open-Shell-Menu#1013] [General]
Last night I found this thread discussing Open-Shell 4.4.170:  
https://www.elevenforum.com/t/open-shell-4-4-170.1047/

Someone had done some work on Open-Shell as late as three months ago.  However, the links given to download this newer version are no longer working.  The version history for 4.4.170 can be found here:  
https://ci.appveyor.com/project/passionate-coder/open-shell-menu/history

I did some searching online and found a number of other sites that offer the 4.4.170 downloaded, but so far, I have not found what I would consider a reliable site to download it from.  So, I have not yet tested this version.

**Comment by Ibuprophen** [comment]
Though this may have reflected that the build was a success, since it wasn't released, the Artifacts (apk build) only remains for a certain period of time. 

These types of builds are only done when certain Pull Requests are submitted and the system creates a build (aka Artifacts) only to see if those Pull Request Changes would (Theoretically) create a successful build. 

I hope I had explained this okay via text... 👍 

~Ibuprophen

**Comment by tallman69** [comment]
I have successfully downloaded and extracted 4.4.170-jgywdlcu as per the last post here:   https://gitter.im/Open-Shell/Lobby?source=orgpage
but have not installed it.

So, Ibuprophen, are you saying that 4.4.170-jgywdlcu is not something that is advisable to install?

**Comment by Ibuprophen** [comment]
That specific one wasn't an official release and the build didn't version itself correctly. That is why the Artifact is no longer available available. 

The specific link to that Korean website was basically a premature post that someone jumped the gun in posting. 

If it's not listed on the Github releases, it's not official. 

This doesn't mean that the project has stopped, it just means that more work is needed. 

Also, keep in mind that some of the team members do utilize software that synchronizes with this Github repository. So those who are working on it this way may not reflect on Github yet until/unless they synchronize with the Github repository. 

~Ibuprophen

**Comment by tallman69** [comment]
Got it.  Thanks.

**Comment by blackcrack** [comment]
humm, 1. "koreanischen Website" 2nd Aug 22, 2021, at this times was Win11 a half Win10, so a alfa/beta Win11 in the meanwhile modified win11 in such a way that OpenShell has a hard time running .. if then, ten only with the explorerpatch from Valinet :  
https://github.com/valinet/ExplorerPatcher/releases/tag/22000.613.44.2_d5f510f  
This version of 4.4.170 is not something special , but if this had builds a hours couples ago with this Versionnummber yea then, but it is not, it's only a older alfa/beta "testbuild" who was deleted imho.. or not deleted, because you have download it @tallman69 , ooor you have lie about the download and have try if there be some officially who watch about this project and have a eye about it.. 
@Ibuprophen .. Ibu, you should looking on the Date on the Posting on the foreign Forum  ;) 
See yea .. or like say the blue tall peoples on an other World(Movie) "I' see you..."

by the way, you can close this Discussion
